client/state/data-layer/wpcom/jetpack-onboarding/test/index.js

```diff
@@ -128,3 +212,14 @@ describe( 'announceSaveFailure()', () => {
 		);
 	} );
 } );
+
+describe( 'fromApi', () => {
+	it( 'should throw an error if no onboarding settings are given', () => {
+		expect( () => fromApi( { data: { noOnboarding: {} } } ) ).toThrow();
+	} );
+	it( 'should return onboarding settings object if present', () => {
+		expect( fromApi( { data: { onboarding: { siteTitle: 'Yet Another Site Title' } } } ) ).toEqual(
```

Comment:
Wondering if we could make these a little more readable by separating `fromApi`'s parameters in separate `const`s.